style/_cards.scss

```diff
@@ -22,6 +22,15 @@
     margin-left: 0;
     margin-right: 0;
   }
+
+  .sd-stretched-link:focus-visible {
+    outline: none;
+
+    &::after {
+      outline-style: auto;
+      outline-color: var(--sd-color-primary);
+    }
```

Comment:
The trick of using `::after` for clickable link-cards has an unfortunate drawback. 

The browser's default stylesheet has some `:focus-visible` rules that help ensure the focus ring is visible. However the `::after` pseudo-element does not receive browser focus. Its parent element, the anchor element, is what receives focus. Therefore the pseudo-element does not match the browser's default `:focus-visible` rules.

This is why I set the outline color here explicitly, because the `::after` pseudo-element will not pick up any of the browser default styles, and without those styles or explicitly setting the outline color, the focus ring color would default to `currentcolor`. When you have a link with white text, this results in a white focus ring, which is not visible for a white-background site.